docs/devel/networking.md

```diff
@@ -99,3 +109,99 @@ As of now, the existent binding mechanisms are:
 - [bridge](#bridge-binding-mechanism)
 - [masquerade](#masquerade-binding-mechanism)
 - [slirp](#slirp-binding-mechanism)
+
+### Bridge binding mechanism
+Using the bridge `BindMechanism` requires a VMI configuration featuring a
+network whose interface type is `bridge` - the yaml file below can be used
+as reference, but please refer to the
+[user guide](https://kubevirt.io/user-guide/#/creation/interfaces-and-networks?id=bridge)
+for more information.
+```yaml
+kind: VM
+spec:
+  domain:
+    devices:
+      interfaces:
+        - name: default
+          bridge: {}
+  networks:
+  - name: default
+    pod: {} # Stock pod network
+```
+
+Let's refer to the image below to get a better understanding of how this
+`BindMechanism` works.
+
+Bridge binding mechanism diagram
```

Comment:
No colon? This is freaking me off. Would you please add one or give me a typography excuse for better sleep?